src/lang-promql/complete/promql.terms.ts

```diff
@@ -18,66 +18,254 @@ export const binOpTerms = [
+  },
+  {
+    label: 'resets',
+    info:
+      'For each input time series, `resets(v range-vector)` returns the number of counter resets within the provided time range as an instant vector. Any decrease in the value between two consecutive samples is interpreted as a counter reset.',
+  },
+  {
+    label: 'round',
+    info:
+      'Rounds the sample values of all elements in `v` to the nearest integer. Ties are resolved by rounding up. The optional `to_nearest` argument allows specifying the nearest multiple to which the sample values should be rounded. This multiple may also be a fraction.',
+  },
+  {
+    label: 'scalar',
+    info:
+      'Given a single-element input vector, `scalar(v instant-vector)` returns the sample value of that single element as a scalar. If the input vector does not have exactly one element, `scalar` will return `NaN`.',
+  },
+  {
+    label: 'sort',
+    info: 'Returns vector elements sorted by their sample values, in ascending order.',
+  },
+  {
+    label: 'sort_desc',
+    info: 'Returns vector elements sorted by their sample values, in descending order.',
+  },
+  {
+    label: 'sqrt',
+    info: 'Calculates the square root of all elements in `v`.',
+  },
+  {
+    label: 'stddev_over_time',
+    info: 'The population standard deviation of the values in the specified interval.',
+  },
+  {
+    label: 'stdvar_over_time',
+    info: 'The population standard variance of the values in the specified interval.',
+  },
+  {
+    label: 'sum_over_time',
+    info: 'The sum of all values in the specified interval.',
+  },
+  {
+    label: 'time',
+    info:
+      'Returns the number of seconds since January 1, 1970 UTC. Note that this does not actually return the current time, but the time at which the expression is to be evaluated.',
+  },
+  {
+    label: 'timestamp',
+  },
+  {
+    label: 'vector',
+    info: 'Returns the scalar `s` as a vector with no labels.',
+  },
+  {
+    label: 'year',
+    info: 'Returns the year for each of the given times in UTC.',
+  },
 ];
 export const aggregateOpTerms = [
-  { label: 'avg' },
-  { label: 'bottomk' },
-  { label: 'count' },
-  { label: 'count_values' },
-  { label: 'group' },
-  { label: 'max' },
-  { label: 'min' },
-  { label: 'quantile' },
-  { label: 'stddev' },
-  { label: 'stdvar' },
-  { label: 'sum' },
-  { label: 'topk' },
+  {
+    label: 'avg',
+    info: 'Calculate the average over dimensions',
+  },
+  {
+    label: 'bottomk',
+    info: 'Smallest k elements by sample value',
+  },
+  {
+    label: 'count',
+    info: 'Count number of elements in the vector',
+  },
+  {
+    label: 'count_values',
+    info: 'Count number of elements with the same value',
+  },
+  {
+    label: 'group',
```

Comment:
`info` is missing here. Is it because you didn't find any documentation ?

Reply:
ah yes I ran a script to generate this from Grafana but they don't have a description for this one. I'll add one manually from Prometheus

Reply:
Added f77b4954f8f560151b19345fb826410ea388e0ba